test/model.jl

```diff
@@ -1001,10 +1001,12 @@ end
     end
 
     @testset "Variable not owned by the model" begin
-        m = Model()
-        M = Model()
-        @variable(m, x)
-        @constraint(M, x == 1)
-        @test_throws ErrorException solve(M)
+        if !isempty(soc_solvers)
+            m = Model()
+            M = Model(solver=first(soc_solvers))
+            @variable(m, x[1:3])
+            @constraint(M, norm(x[1:2]) <= x[3])
+            @test_throws ErrorException solve(M)
```

Comment:
There are lots of errors that can happen on a call to solve. It would be nice to test the specific error message.